sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -215,22 +228,35 @@ public Poller(Duration pollInterval, Function<PollResponse<T>, Mono<PollResponse
      * It will call cancelOperation if status is {@link OperationStatus#IN_PROGRESS} otherwise it does nothing.
      *
      * @throws UnsupportedOperationException when the cancel operation is not supported by the Azure service.
+     * @return A {@link Mono} containing the response.
```

Comment:
Make this clear that this is the poller response (T) and not the final response (R).

Reply:
updated